File: ScannerVB6/ScannerVB6/Model/ModuleBas.cs
namespace ScannerVB6.Model
{
    public class ModuleBas
    {
        public required string NomDuModule { get; set; }
        public required string NomDuFichierBAS { get; set; }
        public int NombreDeUserControles { get; set; } = 0;
        public int NombreDeCreateObject { get; set; } = 0;
        public int NombreDApi32 { get; set; } = 0;
        public int NombreDeExcelApp { get; set; } = 0;
        public int NombreDeWordApp { get; set; } = 0;
        public int NombreDeOfficeouAutre { get; set; } = 0;
        public int NombreDeLignesActives { get; set; } = 0;

        public List<RegexFound> ListofRegeX { get; set; } = new List<RegexFound>();

    }
}

File: ScannerVB/Model/ModuleBas.vb
Public Class ModuleBas
    Property NomDuModule As String
    Property NomDuFichierBAS As String
    Property NombreDeUserControles As Integer = 0
    Property NombreDeCreateObject As Integer = 0
    Property NombreDApi32 As Integer = 0
    Property NombreDeExcelApp As Integer = 0
    Property NombreDeWordApp As Integer = 0
    Property NombreDeOfficeouAutre As Integer = 0
    Property NombreDeLignesActives As Integer = 0

    Property ListofRegeX As New List(Of RegexFound)

End Class
